partiql-tests-data/eval/query/group-by/group-by.ion

```diff
@@ -8377,4 +8377,183 @@
       ]
     }
   },
+  {
+    name:"qualified GROUP BY with qualified SELECT",
+    statement:"SELECT T.col1, COUNT(*) as c FROM simple_1_col_1_group AS T GROUP BY T.col1",
+    assert:{
+      result:EvaluationSuccess,
+      evalMode:[
+        EvalModeCoerce,
+        EvalModeError
+      ],
+      output:$bag::[
+        {
+          col1:1,
+          c:2
+        }
+      ]
+    }
+  },
+  {
+    name:"qualified GROUP BY with qualified SELECT and ORDER BY",
+    statement:"SELECT T.col1, MAX(T.col2) as m FROM simple_2_col_1_group T GROUP BY T.col1 ORDER BY T.col1 DESC",
+    assert:{
+      result:EvaluationSuccess,
+      evalMode:[
+        EvalModeCoerce,
+        EvalModeError
+      ],
+      output:[
+        {
+          m:10,
```

Comment:
Should be 2 not 10? `simple_2_col_1_group` has no 10 value -- https://github.com/partiql/partiql-tests/blob/f1e7a6c534d737fac920d6c4ae7514a34f957239/partiql-tests-data/eval/query/group-by/group-by.ion#L25-L34